airbyte-integrations/connectors/source-instagram/metadata.yaml

```diff
@@ -21,6 +21,13 @@ data:
   releaseStage: generally_available
   releases:
     breakingChanges:
+      3.0.0:
+        message: "Connector will be upgraded to API v18. Customers who use streams `Media Insights`, `Story Insights` or `User Lifetime Insights` must take action with their connections."
```

Comment:
The existing Instagram API (v11) has been deprecated. Customers who use streams `Media Insights`, `Story Insights` or `User Lifetime Insights` must take action with their connections.  Please follow the <migration guide> to update to the latest Instagram API (v18).

Reply:
updated